.travis.yml

```diff
@@ -9,12 +9,24 @@ php:
   - 7.0
   - 7.1
   - nightly
-  # test against the HHVM version provided by travis by default
-  - hhvm
 
 matrix:
   fast_finish: true
   include:
+    # Test against HHVM 3.12 LTS version by using trusty
+    - php: hhvm-3.12
+      sudo: true
+      dist: trusty
+      group: edge # Use edge image until the next travis CI image update
+      addons:
+        apt:
+          packages:
+            - mysql-server-5.6
+            - mysql-client-core-5.6
+            - mysql-client-5.6
+      services:
+        - mysql
```

Comment:
@cebe HHVM did not have a PostgreSQL or PDO-PostgreSQL module until hhvm 3.15

until we added the current hhvm (3.15) there was no PostgreSQL testing in hhvm